javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/TracerInstaller.java

```diff
@@ -127,9 +127,8 @@ private static SpanExporterFactory findSpanExporterFactory(String exporterName)
       if (spanExporterFactory
           .getClass()
           .getSimpleName()
-          .replace("_", "")
           .toLowerCase()
-          .startsWith(exporterName.toLowerCase())) {
+          .startsWith(exporterName.replace("_", "").toLowerCase())) {
```

Comment:
We need to fix that black magic exporter matching soon...

Can you add the same fix to `MetricExporterFactory` and `MetricServer` methods?

Thanks!

Reply:
can we make it so that `otlpspan` does not work? I think better to be strict about only supporting the values in the spec https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-environment-variables.md#exporter-selection

Reply:
done

Reply:
i think `otlpspan` will still work with latest change?

Reply:
@trask, yes, `otlpspan` still works. If I will check for strict list of supported values then how user will be able to specify custom exporter?

Reply:
ok, makes sense, we can tackle this later in #1287